.github/dependabot.yml

```diff
@@ -10,3 +10,8 @@ updates:
     schedule:
       interval: "weekly"
     rebase-strategy: "disabled"
+  - package-ecosystem: "docker"
+    directory: "/docker"
+    schedule:
+      interval: "weekly"
+    rebase-strategy: "disabled"
```

Comment:
So if it tries to merge and can't, it just fails?

Reply:
Honestly, I left this like this because that's how the rest of our `dependabot` config works. I [looked it up](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#rebase-strategy) and all this does is prevent `dependabot` from continuously rebasing the temp branch onto the target branch. So basically we need to hit the "Update branch" button.